kernel/runtime/crt-init/runtime/start.S

```diff
@@ -53,10 +53,10 @@ __dso_handle:
 .skip  8
 .hidden __dso_handle
 
-.section .init
-.global _init
-.type _init,@function
-_init:
+.section .bootlower
```

Comment:
no, not here. this is userspace stuff where the init sections actually belongs to.